week0002/hienhpss/python/week0002.py

```diff
@@ -0,0 +1,37 @@
+def spi_round(n=4,m=4,x=0):
+	'''Looping through the spiral path to return the next
+	cell'''
+	if (x >= (n+1)/2) or (x >= (m+1)/2):
+		return
+	for i in range(x,m-x-1):
```

Comment:
Spacing please. All operators: `+ - * / % == , =` should have space around them, except `=` in function argument passing.